app/attributes/type/gbp.rb

```diff
@@ -0,0 +1,19 @@
+module Type
+  class Gbp < ActiveModel::Type::Decimal
+    def cast(value)
+      if suitable_for_casting?(value)
+        # When a monetary value is provided by a form submission, we are only interested
+        # in the value to 2 decimal places, and discard any more fine detail immediately
+        super(value&.to_s&.delete(',')&.delete('£'))&.round(2)
```

Comment:
I don't like this as the following would be valid:
```
',23,23£23'
```

Reply:
This is straight out of provider app! but will take a look

Reply:
Yeah, this came from CCQ. Our reasoning was that the problem we were trying to solve was (a) accepting and parsing reasonably-formatted values, such as ones with commas and pound signs, while (b) offering fast feedback if the user has typo'd or misunderstood the question, and not silently converting what the user entered into something they didn't intend. We're not trying to _restrict_ users with our validations, we're trying to _help_ them. So given that no one is going to enter `,23,23£23` via typo or misunderstanding the question, there's not a big benefit to validating a case like that, but there _is_ a cost, namely the cost of me having to write a regex, which I'm not very good at...